dbcrossbarlib/src/drivers/gs/prepare_as_destination.rs

```diff
@@ -0,0 +1,44 @@
+//! Preparing bucket directories as output destinations.
+
+use std::process::Command;
+use tokio_process::CommandExt;
+
+use crate::common::*;
+
+/// Prepare the target of this locator for use as a destination.
+pub(crate) async fn prepare_as_destination_helper(
+    ctx: Context,
+    gs_url: Url,
+    if_exists: IfExists,
+) -> Result<()> {
```

Comment:
This deletes any existing data in a `gs://` directory if `IfExists::Overwrite`, before we put data there. Factored out of somewhere else.